gradle/posix.gradle

```diff
@@ -104,3 +104,21 @@ kotlin {
 }
 
 project.ext.set("hasNative", true)
+
+// hack for 1.4-M2, remove after
+// see https://youtrack.jetbrains.com/issue/KT-39302
```

Comment:
Please, add a TODO here to remove this after M3 since it's fixed there according to the issue link

Reply:
The issue is fixed, I try to remove.